# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/ipc/WebrtcProxyChannelCallback.h
# Commit: c684f79d8c67
# Full Hash: c684f79d8c673f21179fce00358d70845a9692c0
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Rename PWebrtcProxyChannel to PWebrtcTCPSocket. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45098
# ==============================================================================

diff -r 82af7b115f60 -r c684f79d8c67 media/mtransport/ipc/WebrtcProxyChannelCallback.h
--- a/media/mtransport/ipc/WebrtcProxyChannelCallback.h	Tue Sep 17 18:15:23 2019 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,30 +0,0 @@
-/* -*- Mode: C++; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
-/* vim: set sw=2 ts=8 et tw=80 ft=cpp : */
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#ifndef webrtc_proxy_channel_callback_h__
-#define webrtc_proxy_channel_callback_h__
-
-#include "nsTArray.h"
-
-namespace mozilla {
-namespace net {
-
-class WebrtcProxyChannelCallback {
- public:
-  NS_INLINE_DECL_PURE_VIRTUAL_REFCOUNTING
-
-  virtual void OnClose(nsresult aReason) = 0;
-  virtual void OnConnected() = 0;
-  virtual void OnRead(nsTArray<uint8_t>&& aReadData) = 0;
-
- protected:
-  virtual ~WebrtcProxyChannelCallback() = default;
-};
-
-}  // namespace net
-}  // namespace mozilla
-
-#endif  // webrtc_proxy_channel_callback_h__